app/main.py

```diff
@@ -1 +1,20 @@
-# write your code here
+def create_report(data_file_name: str, report_file_name: str):
+    newls = []
+    with open(data_file_name) as f:
+        ls = f.readlines()
+    if len(ls[-1]) == 0:
+        ls.pop(-1)
+    for i in ls:
+        newls.append(i.split(","))
```

Comment:
This part of the code may be not good because you already have the list from "ls = f.readlines()" and during the loop you can sort it like "i.split(",")" after that put correctly values in loop.You should call clearer names of variable, not use "i, ls, f"